projects/desktop/src/assets/locales/NL.json

```diff
@@ -44,6 +44,10 @@
       }
     },
 
+    "onboarding": {
+      "title": "Je bent er <span>bijna</span>!"
+    },
```

Comment:
Why there is not this part:
"form": {
"nickname": "Pseudo",
"birthdate": "Date de naissance",
"finish": "C'est parti !"
}